src/test/resources/expected_helm_output/bitbucket/output.yaml

```diff
@@ -28,6 +28,72 @@ data:
   max_heap: 1g
   min_heap: 512m
 ---
+# Source: bitbucket/templates/service-mesh.yaml
```

Comment:
This was recorded to make the tests passing but will need to be recorded again after we do more changes.